### DISCUSSION TITLE
Captcha Issue

### DISCUSSION THREAD

**Discussion by rezytijo** [vogler/free-games-claimer#284] [Q&A]
Any suggestions for resolving captcha?  I deployed it on a VPS so it was difficult to get a new IP or avoid captchas

**Comment by GalSasson6** [comment]
ive ran into same issue

**Comment by Sharpyku** [comment]
Everyone has encountered this issue, that's why free-games-claimer is becoming almost useless. I don't know what you want to use it for, but if you're using it on your own PC, it's easy to get past the captcha—you just complete it manually and that's it. The problem is when you want to use it through a VPS, it becomes really complicated. I managed to create a dashboard where I log into Epic Games, copy the cookies, and import them into a file, and that way it stays logged in—but only for a short period, max 6-7 days, then I get logged out and have to update them each time.

You can do the same on your PC, but it's totally pointless. There are Discord servers that notify you when there are free games, and you claim them manually—it's the same thing. Personally, I gave up, and I think Epic Games Claimer is RIP

**Comment by feldorn** [comment]
You should check out the technique used here: https://claabs.github.io/epicgames-freegames-node/index.html
It works pretty well for me, but would be ideal to be incorporated into what you've done for other sites.
